.github/workflows/ci.yml

```diff
@@ -191,7 +191,7 @@ jobs:
       fail-fast: false
       matrix:
         node-version: [14, 18]
-        postgres-version: [9.5, 10]
+        postgres-version: [10, 12]
```

Comment:
It should.  My understanding of GH workflows (and Matrix) is weak.  I thought the workflow would only run the listed version -- if I place `14` does it have its own instance of Postgres 14?

I only looked for database containers by the previous author to keep the scripts simple and because I'm also not a Sequelize/Docker expert yet -- I'm uncertain what configurations he did that differ from the official Postgres images.  I didn't find a Postgres 14 container by the same author, I only saw up to v12.  My thinking here was v14 could be added at a later point.

I imagine in an ideal world, each database version Sequelize supports would be included, but these tests are lengthy and it's good to limit.  For that reason, I submit that the minimum and latest versions should definitely be included and an optional popular database in between.

Reply:
Yeah, we need to implement our own Docker scripts for postgres 14 so we're not dependent on Sushant. Let's add the support for postgres 12 in one PR and later one for postgres 14